core-bundle/src/Filesystem/FilesystemUtil.php

```diff
@@ -12,12 +12,36 @@
 
 namespace Contao\CoreBundle\Filesystem;
 
+use Contao\CoreBundle\Filesystem\Dbafs\UnableToResolveUuidException;
+use Contao\StringUtil;
+use Symfony\Component\Uid\Uuid;
+
 /**
- * @internal
+ * @experimental
  */
 class FilesystemUtil
 {
     /**
+     * Gets all files from a serialized string or array of binary UUIDs like
+     * for instance stored in "tl_content.multiSRC".
+     *
+     * The following rules apply:
+     *  - Invalid or non-existent UUIDs are skipped without producing an error.
+     *  - If the UUID points to a directory, its file contents are used (only first level).
+     *  - Duplicate files are stripped.
+     *
+     * @param string|array<string|null> $sources
+     */
+    public static function listContentsFromMultiSRC(VirtualFilesystemInterface $storage, string|array $sources): FilesystemItemIterator
```

Comment:
```suggestion
    public static function listContentsFromSerialized(VirtualFilesystemInterface $storage, string|array $sources): FilesystemItemIterator
```

Because maybe in the future we have a json storage. `MultiSRC` is not the definition. The format is. Which is serialized in this case :)

Reply:
see fc7284031d3c9959c9cce4e3559a3427a7c2c30e